config/locales/en.yml

```diff
@@ -6,12 +6,12 @@ en:
     home:
       datasets:
         header: Geospatial data
-        description: Need copy for this ... contains over %{sets} datasets...
+        description: Data that includes location information (coordinates) and is natively usable in GIS software and services such as ArcGIS, QGIS, WFS, and WMS. Geospatial data types include vector data, such as Shapefile, GeoJSON, and GeoPackage, and raster data, such as GeoTIFF, grid files, and georeferenced maps.
       maps:
         header:  Scanned maps
-        description: Need copy for this ... contains over %{sets} maps...
+        description: Images of paper maps that have been scanned and/or georeferenced. These digital cartographic resources cover all parts of the world and include topographical maps, historical maps, nautical charts, thematic maps, and atlases.
```

Comment:
If we are no longer using sets, we should update the code to not calculate the number of records.

Reply:
Hm. Let's confirm we won't want that anywhere? I will add as a standup topic. Good idea!